build/buildconstants/params_mainnet.go

```diff
@@ -23,8 +22,7 @@ var DrandSchedule = map[abi.ChainEpoch]DrandEnum{
 
 var NetworkBundle = "mainnet"
 
-// NOTE: DO NOT change this unless you REALLY know what you're doing. This is consensus critical.
-var BundleOverrides map[actorstypes.Version]string
```

Comment:
I'm pretty unfamiliar with this code so could you please
1. Explain why we are removing this
2. Explain why it is not consensus breaking

Reply:
ah ok I see it now: https://github.com/filecoin-project/lotus/pull/12112#discussion_r1675022085

Reply:
> I'm pretty unfamiliar with this code so could you please
> 1. Explain why we are removing this
> 2. Explain why it is not consensus breaking

I've moved constants to the buildconstsnts package and put deprecated refs here. Nothing is removed. 
Therefore, concensus is unchanged. 

This is part of the effort to make a usable API for Curio that doesn't import lotus internals.